src/btreemap/proptests.rs

```diff
@@ -34,6 +36,8 @@ fn operation_strategy() -> impl Strategy<Value = Operation> {
         15 => (any::<usize>()).prop_map(Operation::Remove),
         5 => (any::<usize>(), any::<usize>())
             .prop_map(|(from, len)| Operation::Range { from, len }),
+        15 =>  Just(Operation::PopFirst),
+        15 =>  Just(Operation::PopLast),
```

Comment:
nit: I'd give these smaller weights. We want the btree in this test grow over time, and putting large weights on removing elements will keep the btree too small to have meaningful tests.

```suggestion
        2 =>  Just(Operation::PopFirst),
        2 =>  Just(Operation::PopLast),
```